tests/pass-dep/tokio/tokio_mvp.rs

```diff
@@ -1,5 +1,5 @@
 // Need to disable preemption to stay on the supported MVP codepath in mio.
-//@compile-flags: -Zmiri-disable-isolation -Zmiri-permissive-provenance -Zmiri-preemption-rate=0
+//@compile-flags: -Zmiri-disable-isolation -Zmiri-permissive-provenance
```

Comment:
Removing the `-Zmiri-preemption-rate=0` here was premature, this is causing issues again. I'll add the flag back.